uaclient/entitlements/tests/test_esm.py

```diff
@@ -184,10 +184,26 @@ def test_esm_apps_disable_apt_auth_only_is_true_on_lts(
 
 @mock.patch("uaclient.entitlements.esm.update_ua_messages")
 class TestESMInfraEntitlementEnable:
+    @pytest.mark.parametrize(
+        "esm_cls", [ESMAppsEntitlement, ESMInfraEntitlement]
+    )
+    @mock.patch("uaclient.apt.setup_apt_proxy")
```

Comment:
Just another nit. We could add this patch into the class definition, like `update_ua_messages` just to avoid repetition.